### PR TITLE
Fix nil map

### PR DESCRIPTION
**Pull request by tzwsoho**
修正map没有初始化而直接使用时报assignment to entry in nil map错误的问题

### CODE REVIEW

File: tars/tools/tars2go/gen_go.go [hunk 1]
```diff
@@ -1413,7 +1413,13 @@ func (gen *GenGo) genSwitchCase(tname string, fun *FunInfo) {
 	inArgsCount := 0
 	outArgsCount := 0
 	for _, v := range fun.Args {
-		c.WriteString("var " + v.Name + " " + gen.genType(v.Type) + "\n")
+		c.WriteString("var " + v.Name + " " + gen.genType(v.Type))
+		switch v.Type.Type {
+		case tkTMap:
+			c.WriteString(" = make(map[" + gen.genType(v.Type.TypeK) + "]" + gen.genType(v.Type.TypeV) + ")\n")
```
Comment (chenhengqi) [inline tars/tools/tars2go/gen_go.go:1419]: Consider replace this with `gen.genType(v.Type)`.

**Review by chenhengqi — CHANGES_REQUESTED** [review]

**Comment by chenhengqi** [comment]
Squash your commits please.

**Comment by tzwsoho** [comment]
> Squash your commits please.

Done

**Comment by chenhengqi** [comment]
> > Squash your commits please.
> 
> Done

I mean you should combine all the commits into one, they are not relative.

**Pull request closed without merging**